test/unit/multiplication/test_multiplication_general.cpp

```diff
@@ -162,7 +251,9 @@ std::vector<GemmConfig> sub_gemm_configs = {
     // single-tile
     {blas::Op::NoTrans, blas::Op::NoTrans, 8, 8, 11, 10, 9, 13, {{2, 1}}, {{1, 1}}, {{0, 0}}},
     // multi-tile
-    {blas::Op::NoTrans, blas::Op::NoTrans, 12, 20, 11, 3, 4, 5, {{7, 1}}, {{11, 10}}, {{4, 2}}},
+    // TODO check this, I suspect a problem/limit with panel offset
```

Comment:
note-to-self: after a quick check it seems a limitation of panel. it needs more investigation

Reply:
See #1047